x/stakeibc/client/cli/tx.go

```diff
@@ -40,7 +40,6 @@ func GetTxCmd() *cobra.Command {
 	cmd.AddCommand(CmdUndelegateHost())
 	cmd.AddCommand(CmdUpdateInnerRedemptionRateBounds())
 	cmd.AddCommand(CmdResumeHostZone())
-	cmd.AddCommand(CmdCreateTradeRoute())
```

Comment:
So it is still part of the "API" as a tx route but it isn't part of the CLI? Makes sense I hadn't seen an example of this before but of course those are separable

Reply:
yup that's right! cause you initiate it through the CLI via the `submit-proposal` tx in the gov module instead of through stakeibc (code above)